packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -135,14 +135,12 @@ export class VaRadio {
 
     switch (event.key) {
```

Comment:
Jamie Klenetsky Fay & Rob Garrison suggested in the ticket that we remove this switch statement, however I found that the functionality (other than the `preventDefault()`) does not affect the input fields and only updates attributes on the web-component. I'm hoping that removing the `preventDefault` on the event will solve the issue with JAWS.